lib/models/repository-states/present.js

```diff
@@ -258,8 +257,8 @@ export default class Present extends State {
   // Remote interactions
 
   @invalidate(branchName => [
-    Keys.aheadCount.oneWith(branchName),
-    Keys.behindCount.oneWith(branchName),
+    Keys.statusBundle,
+    Keys.headDescription,
```

Comment:
If the nearest tag reachable from a commit while in detached head state is `origin/branch-name`, then a fetch or pull that updates a remote tracking branch could affect the commit description